packages/amplify-category-geo/src/service-stacks/baseStack.ts

```diff
@@ -42,9 +42,13 @@ export class BaseStack extends cdk.Stack {
    * converts to cfn
    */
   toCloudFormation = (): $TSAny => {
-    cdkV1PrepareAppShim(this);
-    const cfn = this._toCloudFormation();
-    return cfn;
+    const root = this.node.root as cdk.Stage;
+    const assembly = root.synth();
+    if (!this.nestedStackParent) {
+      return assembly.getStackArtifact(this.artifactId).template;
+    }
+    // if this is a nested stack ( i.e. it has a parent), then just read the template as a string
+    return JSON.parse(fs.readFileSync(path.join(assembly.directory, this.templateFile)).toString('utf-8'));
```

Comment:
nit: I'd prefer this broken up for readability

```suggestion
    const template = fs.readFileSync(path.join(assembly.directory, this.templateFile);
    return JSON.parse(template).toString('utf-8'));
```